lib/src/services/arp_service.dart

```diff
@@ -1,6 +1,14 @@
 import 'package:network_tools/src/models/arp_data.dart';
 
 abstract class ARPService {
+  ARPService() {
+    _instance = this;
+  }
+
+  static late ARPService _instance;
+
+  static ARPService get instance => _instance;
+
```

Comment:
what kind of hack you did here @guyluz11 ? how can dart allow `this` keyword here?

Reply:
`network_tools_flutter` tests are failing